### DISCUSSION TITLE
New DBeaver install - how to set up first PostgreSQL database

### DISCUSSION THREAD

**Discussion by devilkepo74** [dbeaver/dbeaver#11226] [Q&A]
I just installed DBeaver on my personal computer and am trying to create a PostgreSQL database.

I entered the following in the Connect to database window:

Host: localhost

Port: 5432

Database: postgres
User: postgres

Those values were all pre-filled defaults, so I left them and clicked t he Test Connection... button.

When I click the Test button an error window opens, saying, "Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections. Connection refused: connect"

I tried setting my own db name and credentials, but get the same results.

How do I set up a local PostgreSQL database in DBeaver?

**Comment by flambchap** [comment]
Repost from Stack Overflow, watch out for spam link injection.
